apps/emqx_plugin_libs/src/emqx_trace/emqx_trace.erl

```diff
@@ -186,6 +188,14 @@ trace_file(File) ->
             {error, Node, Reason}
     end.
 
+trace_file_detail(File) ->
+    FileName = filename:join(trace_dir(), File),
+    Node = atom_to_binary(node()),
+    case file:read_file_info(FileName, [{'time', 'posix'}]) of
+        {ok, #file_info{size = Size, mtime = Mtime}} -> {ok, Node, #{size => Size, mtime => Mtime}};
```

Comment:
nit: why not return {ok, #{node => Node, size => Size, mtime => Mtime}}

Reply:
HA， On the other hand, I think to keep the same tuple for error and ok is fine too.